tests/test_mesh.py

```diff
@@ -51,25 +44,31 @@ def assert_includes(dicts, dict_cores):
 
 
 @mock.patch('galter_subjects_utils.downloader.requests.get')
-def test_downloader(patched_get):
+def test_downloader(patched_get, tmp_path):
+    downloads_dir = tmp_path / "downloads"
+    downloads_dir.mkdir()
     # patch requests.get to return files
     patched_get.side_effect = fake_request_context
-    downloads_dir = Path(__file__).parent / "downloads"
+
+    # Testing the current_year variant seemed like it would lead to a test
+    # mirror, so it has been tested manually instead. This really just tests
+    # that we have the right name, interfaces and final files written to disk
+    # in right location.
     downloader = MeSHDownloader(
-        year="2022",
-        prefixes=["d", "q"],
+        year="2023",
         directory=downloads_dir
     )
-
     downloader.download()
 
     patched_get.assert_called()
-    assert downloads_dir / "d2022.bin" == downloader.prefix_to_filepath["d"]
-    assert downloads_dir / "q2022.bin" == downloader.prefix_to_filepath["q"]
+    assert (downloads_dir / "d2023.bin").exists()
```

Comment:
Ha, took me a second to realize these are **2023** which made it clear what was going on 😆